### DISCUSSION TITLE
What's going on with the Play store version?

### DISCUSSION THREAD

**Discussion by LucasParsy** [spacecowboy/Feeder#351] [General]
My feeder app seemed to stop working a few days ago, then I couldn't update it via the Play store, as it doesn't want to show me the page for the app.

In the main time I started getting captchas and cloudflare anti bot screens on my phone web browsing, and I checked my phone data and Feeder seemed to have used lots of data in the background recently... 
What's happening ? :(

**Comment by spacecowboy** [comment]
There hasn't been a release for a long while now so that's why there is nothing to update.

As for the background data, others have reported spikes but I still have no idea what they are.

**Comment by LucasParsy** [comment]
All my excuses, there was a problem with the cache of my play store that made it impossible to update and access the page of specifically the Feeder app.
Other problems I described are most probably unrelated.
Sorry for doubting you!
